### PR TITLE
Add AD and Alerting to the manifest 2.0.0 and remove CU/Alerting build script

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Add AD and Alerting to the manifest 2.0.0 and remove CU/Alerting build script

Corresponding changes:
https://github.com/opensearch-project/common-utils/pull/152
https://github.com/opensearch-project/alerting/pull/377
 
### Issues Resolved
Part of #1624
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.0.0/opensearch-2.0.0.yml
Comment (peterzhuamazon) [reply manifests/2.0.0/opensearch-2.0.0.yml]: fixed now.
Thanks.

**Review by zelinh — APPROVED** [review]
Thanks!

**Comment by peterzhuamazon** [comment]
Hold this for a bit before merge.

**Review by zelinh — APPROVED** [review]
Thanks!

**Pull request merged**
